src/parsers/manifestjson.js

```diff
@@ -108,7 +110,14 @@ export default class ManifestJSONParser extends JSONParser {
     // Not all messages returned by the schema are fatal to Firefox, messages
     // that are just warnings should be added to this array.
     const warnings = [messages.MANIFEST_PERMISSIONS.code];
-    const validate = this.isLanguagePack ? validateLangPack : validateAddon;
+    const isStaticTheme = Object.prototype.hasOwnProperty.call(this.parsedJSON, 'theme');
```

Comment:
I don't know about the code style in addons-linter... but if this was addons-server I'd be asking should this be set as a  property on the object, like `isLanguagePack` (and maybe in the same place, rather than in `_validate`)?

Reply:
Hmm, I'm torn here, but might be a good idea.